### ISSUE TITLE
AVT 1 - Icon: DAP violations - undefined role

### ISSUE THREAD

**Issue by snidersd** [IBM/carbon-components-react#2232]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
Run DAP on the icon component
See screenshot below for violations:
<img width="1244" alt="Screen Shot 2019-04-17 at 3 08 02 PM" src="https://user-images.githubusercontent.com/21676914/56317812-4ae85300-6123-11e9-88b1-462da8d19246.png">

Note: Add a valid ARIA role or remove the role entirely if it is not needed.

**Comment by snidersd** [comment]
This issue should remain open until it is fixed and retested.

**Comment by dakahn** [comment]
Should this SVG have a `role="img"` attribute? @snidersd
